add heading for creating web app client secret

spottedmahn opened the pull request:
this will allow for a direct link to creating the client secret.  The client secret is referenced in various samples like [active-directory-b2c-devquickstarts-web-api-dotnet](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-devquickstarts-web-api-dotnet) but there is no link to it and how to set it up.

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

arramac commented:
Ack

spottedmahn committed:
add heading for creating web app client secret

this will allow for a direct link to creating the client secret.  The client secret is referenced in various samples like [active-directory-b2c-devquickstarts-web-api-dotnet](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-devquickstarts-web-api-dotnet) but there is no link to it and how to set it up.
